typescript-tests/object.ts

```diff
@@ -1,12 +1,22 @@
 import { normalize, schema } from '../index'
 
-const data = {
-  /* ...*/
-};
+type Response = {
+  users: Array<{ id: string }>
+}
+const data: Response = { users: [ { id: 'foo' } ] };
 const user = new schema.Entity('users');
 
-const responseSchema = new schema.Object({ users: new schema.Array(user) });
-const normalizedData = normalize(data, responseSchema);
+{
```

Comment:
What do these added brackets do?

Reply:
They provide unique block scope for each of the tests, so I can redeclare variables using the same name.